messaging/src/main/java/org/axonframework/messaging/eventhandling/annotation/MethodSequencingPolicyEventHandlerDefinition.java

```diff
@@ -45,6 +46,9 @@ public class MethodSequencingPolicyEventHandlerDefinition implements HandlerEnha
 
     @Override
     public @Nonnull <T> MessageHandlingMember<T> wrapHandler(@Nonnull MessageHandlingMember<T> original) {
+        if (original.attribute(HandlerAttributes.EVENT_NAME).isEmpty()) {
```

Comment:
Curious: did you include this ensure event handlers weren't invoked for `ResetContext` messages?